api/models/orderDao.js

```diff
@@ -0,0 +1,62 @@
+const { appDataSource } = require("../database/database");
+
+
+async function createOrderTable(userId, totalCost) {
+    try {
+			const orderTable = await appDataSource.query(
+				`
+					INSERT INTO orders 
+						(user_id, total_cost) 
+					VALUES (?, ?)
+    		`, [ userId, totalCost ]);
+
+    return orderTable;
+        
+    } catch(err) {
+        console.log("ORDER DAO1")
+        throw err;
+    } 
+}
+
+async function createOrdersRequest(orderId, products) {
+    try {
+        const query = 
+				`
+					INSERT INTO order_product 
+						(orders_id, product_id, quantity)
+        	VALUES ?;
+        `
+        const values = products.map((product) => {
+            return [ orderId, product.id, product.quantity ];
+        });
+    
+        const requestResult = await appDataSource.query(query, [values]);
+
+        return requestResult;
```

Comment:
```suggestion
        const values = products.map((product) => {
            return [ orderId, product.id, product.quantity ];
        });
    
        return appDataSource.query(`
	  INSERT INTO order_product 
	    (
	      orders_id, 
	      product_id, 
	      quantity
	    ) VALUES ?;
        `, [values]);
```
보다 보기 좋고 간단하게 쓰일 수 있을 것 같습니다.